wormdatamodel/signal/signal.py

```diff
@@ -12,9 +11,7 @@
 from scipy.io import loadmat as sioloadmat
 from copy import deepcopy as deepcopy
 from datetime import datetime
-import mistofrutta.struct.irrarray as irrarray
 import wormdatamodel as wormdm
-import savitzkygolay as sg
```

Comment:
I moved each of these into the scopes of the functions that use them so someone can import the `wormdatamodel` package without needing to find each GitHub repo and do local installs

Using each function would still error if the package is missing, but at least that only occurs when (and 'if', in my case) those functions would be called

Reply:
An alternative to this, if you prefer to keep things as they are, might be to include git-based installations in the setup requirements for this package

That could however cascade a series of PRs like this one if there's any trouble installing those packages however